src/index.html

```diff
@@ -4,9 +4,383 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#" class="header__logo-link">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a
+              href="#computers"
+              class="nav__link"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tabs" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="main-container">
+      <main class="main">
+        <section class="prd-card main__prd-card-container" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="apple a1419 imac 27 retina 5K"
+            class="prd-card__image"
+          >
+          <h1 class="prd-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
```

Comment:
There should be only one `h1` element per page. Use `h2`

Reply:
Got you, thank you :)